projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -145,6 +147,38 @@ public static function init_conditional_recommendation_actions() {
 
 		// Monitor for changes in plugins that have auto-updates enabled
 		add_action( 'update_site_option_auto_update_plugins', array( static::class, 'plugin_auto_update_settings_changed' ), 10, 3 );
+
+		// Subscriber count check. This runs when stats are fetched.
+		add_action( 'jetpack_followers_fetched', array( static::class, 'recommend_paid_newsletter' ), 10, 2 );
+	}
+
+	/**
+	 * Checks a sites subscriber count and triggers the recommendation if it is over 100 or under 100k.
+	 *
+	 * @param array $followers Array of followers data from WPcom stats check.
+	 * @link https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/stats/followers/
+	 */
+	public static function recommend_paid_newsletter( $followers ) {
+		// return if followers is not an array.
+		if ( ! is_array( $followers ) ) {
+			return;
+		}
+
+		// return if the subscriptions module is not active
+		if ( ! Jetpack::is_module_active( 'subscriptions' ) ) {
+			return;
+		}
+
+		// if followers array does not have total key, return.
+		if ( array_key_exists( 'total', $followers ) ) {
+			// the endpoint used in class-wpocom-stats does not pass type=all as a parameter. Summing the parts gives the total for this purpose
+			$total_subscribers = $followers['total_email'] + $followers['total_wpcom'];
+			if ( $total_subscribers >= 100 && $total_subscribers < 100000 ) {
```

Comment:
For future tests, I think we can just temporarily edit this line to change the requirements to enable the recommendation.

Reply:
Yeah, set it to 101 :)